cmd/testutils/extractor/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func main() {
+	network := flag.String("n", "", "network name.")
+	filePath := flag.String("p", "", "file path.")
+	flag.Parse()
+
+	bytes, err := os.ReadFile(*filePath)
+	if err != nil {
+		fmt.Println("Error in resolving ", err)
```

Comment:
Probably should exit here with a `panic`, `log.Fatal` or `os.Exit(1)` instead of proceeding.